src/plugins/auto/src/cumulative_schedule.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include "schedule.hpp"
+#include "async_infer_request.hpp"
+
+#ifdef  MULTIUNITTEST
+#define MOCKTESTMACRO virtual
```

Comment:
Why should plugin code have any definitions for tests?
Can we avoid a duplication of these definitions?

Reply:
updated